go/vt/topo/tablet.go

```diff
@@ -234,8 +230,6 @@ func (ts *Server) GetTabletAliasesByCell(ctx context.Context, cell string) ([]*t
 // GetTabletsByCellOptions controls the behavior of
 // Server.FindAllShardsInKeyspace.
 type GetTabletsByCellOptions struct {
-	// Concurrency controls the maximum number of concurrent calls to GetTablet.
-	Concurrency int
```

Comment:
This could be kept if we want to be able to have a limit that is below `DefaultReadConcurrency`

Reply:
We might want to retain this, but we can add it back if there's a compelling use case.